build/steps/prepare-build.yml

```diff
@@ -3,7 +3,13 @@ parameters:
 
 steps:
 - task: UseDotNet@2
-  displayName: Use .NET 3.1
+  displayName: Ensure 3.1 SDK is Installed
+  inputs:
+    packageType: 'sdk'
+    version: '3.1.404'
```

Comment:
Should this be changed to '5.0.x'

It looks like Azure DevOpe Pipelines does support .net5.0 apps now:
[https://stackoverflow.com/questions/64830501/building-net-5-0-project-azure-devops-pipeline](https://stackoverflow.com/questions/64830501/building-net-5-0-project-azure-devops-pipeline)

Reply:
@calebnelton you may notice that we use a global.json, the hardcoded 3.1 sdk is a hack for Azure DevOps